File: Java/백준/Bronze/28097. 모범생 포닉스/모범생 포닉스.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int N = Integer.parseInt(br.readLine()); //공부 계획의 수

        int sum = 0;
        String[] input = br.readLine().split(" ");

        for (int i = 0; i < N; i++) {
            int study = Integer.parseInt(input[i]);
            sum += study;
        }

        sum += 8 * (N - 1);

        int day = sum / 24;
        int hour = sum % 24;

        System.out.print(day + " " + hour);
    }
}

File: C11/백준/Bronze/28097. 모범생 포닉스/모범생 포닉스.c
#include <stdio.h>

int main(void) {
	
	int N;
	scanf("%d", &N);
	
	int sum = 0;
	for(int i = 0; i < N; i++) {
		int study;
		scanf("%d", &study);
		sum += study;
	}
	
	sum += 8 * (N - 1);
	
	int day = sum / 24;
	int hour = sum % 24;
	
	printf("%d %d", day, hour);
	
	return 0;
}